add-chain/utils.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+func executeCommand(command string, args []string) (string, error) {
+	cmd := exec.Command(command, args...)
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	return out.String(), err
+}
+
+func inferRpcUrl(superchainTarget string) (string, error) {
+	switch superchainTarget {
+	case "mainnet":
+		return "https://ethereum-mainnet-rpc.allthatnode.com", nil
+	case "sepolia":
+		return "https://ethereum-sepolia-rpc.allthatnode.com", nil
+	}
+	return "", fmt.Errorf("unsupported Superchain Target %s", superchainTarget)
+}
```

Comment:
We should get these RPC urls from this file:

https://github.com/ethereum-optimism/superchain-registry/blob/f56b630610ea41bb7e23d262de6f7cbb897d6ca4/superchain/configs/mainnet/superchain.yaml#L4-L5

Reply:
Fixed here: https://github.com/ethereum-optimism/superchain-registry/pull/204/commits/7205dbe433e9dcf31c3376e6104fb1f08e9c62ed